docs/other.rst

```diff
@@ -141,3 +141,38 @@ Lego Port
 
 .. autoclass:: ev3dev2.port.LegoPort
     :members:
+
+Wheels
+------
+All Wheel class units are in millimeters.  The ``diameter_mm`` and ``width_mm`` of the wheel must be specified whenever a new child Wheel class is added to ``ev3dev2/wheel.py``. The diameter and width for various lego wheels can be found at http://wheels.sariel.pl/
```

Comment:
Isn't the sentence about `wheel.py` an implementation detail that users don't need to know about?

Reply:
will chop that sentence